### DISCUSSION TITLE
Copilot unable to check revocation for the certificate

### DISCUSSION THREAD

**Discussion by harningle** [community/community#41011] [Copilot Conversations] [answered]
### Select Topic Area

Issues

### Body

I'm using copilot behind a proxy and cannot login to github. I ran in command line `curl -x http://my-proxy:port -i -L https://copilot-proxy.githubusercontent.com/_ping` and it says `curl: (35) schannel: next InitializeSecurityContext failed: Unknown error (0x80092012) - The revocation function was unable to check revocation for the certificate.`

If I disable ssl revoke then it seems to work:

```
curl https://copilot-proxy.githubusercontent.com/_ping -x http://my-proxy:port --ssl-no-revoke
{"now":1670423666,"status":"ok"}
```

I'm wondering if there is any solution?

**Comment by ossanna16** [comment]
Hi there @harningle and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by iamcjwilliams** [comment, marked as answer]
@harningle The error message unable to get local issuer certificate means a proxy that uses self-signed certificates is being used. Unfortunately, self-signed certificates is something we currently don't support. We'd recommend that you keep an eye on our [official GitHub public product roadmap](https://github.com/github/roadmap), our [Changelog](https://github.blog/changelog/) and our [GitHub Blog](http://github.blog/) for future updates.

**Reply by harningle** [reply]
Many thanks! I was hoping there were some "dirty" way to bypass that hahaha
